features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/appm/js/publisher.commons.js

```diff
@@ -186,7 +186,7 @@ $(document).ready(function() {
 			success : function(response) {
 				$('#modal-redirect').modal('show');
 				setTimeout(function() {
-					var newVersionDetails = JSON.parse(response);
+					var newVersionDetails = response;
```

Comment:
removed redundant variable `newVersionDetails` in commit 8ff49da4082892971f0bb39e79b15180eb19bcc2 of PR #240